setup.sh

```diff
@@ -0,0 +1,55 @@
+#! /bin/bash
+
+# This script sets up a Ubuntu machine to run this Node.js app locally
+
+
+# Check if node is in apt-cache
+if [ $(apt-cache show nodejs) ]
+then
+  echo "nodejs is already in the apt-cache"
+else
+  sudo apt update
+fi
+
+# Install Node.js runtime environment
+if [ -n "$(nodejs --version)" ]
+then
+  echo "Node.js is already installed"
+else
+  echo "Installing nodejs"
+  curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash -
+  sudo apt install -y nodejs
+fi
+
+# Install git
+if [ -n "$(git --version)" ]
+then
+  echo "git is already installed"
+else
+  echo "Installing git"
+  sudo apt-get update
```

Comment:
The safest thing to do is put an apt-cache conditional on every package we could be installing like node above.

Reply:
We are no longer checking the apt-cache for `nodejs`